app/pages/sections/additional-services/select/additional-service/controller.test.js

```diff
@@ -68,6 +68,31 @@ describe('additional-services select-additional-service controller', () => {
         logger,
       });
     });
+
+    it('should return empty list when getData returns undefined', async () => {
```

Comment:
what if it's returned not as an array, but something else? can it be made resilient to object type change?

Reply:
Apologies, not sure what you mean?